rules/aws_cloudtrail_rules/aws_rds_master_pass_updated.py

```diff
@@ -0,0 +1,18 @@
+from panther_base_helpers import deep_get
+
+
+def rule(event):
+    master_user_pass = deep_get(
```

Comment:
I suggest filtering before deep_get. There's no chance that the deep_get will return results without the eventname and source matching.

Reply:
I believe python automatically will exit execution if the first statement evaluates to be false.